Gems/ROS2/Code/Source/VehicleDynamics/DriveModels/AckermannDriveModel.cpp

```diff
@@ -71,23 +66,21 @@ namespace ROS2::VehicleDynamics
 
     void AckermannDriveModel::ApplyWheelSteering(SteeringDynamicsData& wheelData, float steering, double deltaTimeNs)
     {
-        const double deltaTimeSec = double(deltaTimeNs) / 1e9;
 
         const auto& steeringEntity = wheelData.m_steeringEntity;
-        AZ::Vector3 currentSteeringElementRotation;
-        AZ::TransformBus::EventResult(currentSteeringElementRotation, steeringEntity, &AZ::TransformBus::Events::GetLocalRotation);
-        const float currentSteeringAngle = currentSteeringElementRotation.Dot(wheelData.m_turnAxis);
-        const double pidCommand = m_steeringPid.ComputeCommand(steering - currentSteeringAngle, deltaTimeNs);
-        if (AZ::IsClose(pidCommand, 0.0))
-        {
-            return;
-        }
+        const auto& hingeComponent = wheelData.m_hingeJoint;
+
+        auto id = AZ::EntityComponentIdPair(steeringEntity, hingeComponent);
 
-        const float torque = pidCommand * deltaTimeSec;
-        AZ::Transform steeringElementTransform;
-        AZ::TransformBus::EventResult(steeringElementTransform, steeringEntity, &AZ::TransformBus::Events::GetWorldTM);
-        const auto transformedTorqueVector = steeringElementTransform.TransformVector(wheelData.m_turnAxis * torque);
-        Physics::RigidBodyRequestBus::Event(steeringEntity, &Physics::RigidBodyRequests::ApplyAngularImpulse, transformedTorqueVector);
+        PhysX::JointRequestBus::Event(
+            id,
+            [&](PhysX::JointRequests* joint)
+            {
+                double  currentSteeringAngle = joint->GetPosition();
```

Comment:
nitpick: extra space after the `double` keyword.
```suggestion
                double currentSteeringAngle = joint->GetPosition();
```